src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -207,10 +228,12 @@ export class RelayFeeCalculator {
 
     if (toBN(amountToRelay).eq(0)) return MAX_BIG_INT;
 
-    const getGasCosts = this.queries.getGasCosts(deposit, amountToRelay, relayerAddress).catch((error) => {
-      this.logger.error({ at: "sdk-v2/gasFeePercent", message: "Error while fetching gas costs", error });
-      throw error;
-    });
+    const getGasCosts = this.queries
+      .getGasCosts(deposit, simulateZeroFill ? bnOne : amountToRelay, relayerAddress)
+      .catch((error) => {
+        this.logger.error({ at: "sdk-v2/gasFeePercent", message: "Error while fetching gas costs", error });
```

Comment:
While here, should we include in this log message?
 - deposit information
 - simulateZeroFill flag

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/445/commits/901d590328278ef5e1fb2e7c4aa03e4614ea0823